targets/PythonSdk/source/PlayFab/PlayFabHTTP.py

```diff
@@ -51,18 +54,24 @@ def DoPost(urlPath, request, authKey, authVal, callback, customData = None, extr
         callGlobalErrorHandler(error)
 
         try:
-            callback(None, error) # Notify the caller about an API Call failure
+            # Notify the caller about an API Call failure
```

Comment:
># Notify the caller about an API Call failure [](start = 12, length = 45)

Should these be above or below the line of code, according to Python standard? (sorry, don't know them myself)

Reply:
From my experience they should be above